open-api/rest-catalog-open-api.yaml

```diff
@@ -2603,30 +2603,29 @@ components:
       properties:
         type:
           type: "string"
+          enum:
+          - "assert-create"
+          - "assert-table-uuid"
+          - "assert-ref-snapshot-id"
+          - "assert-last-assigned-field-id"
+          - "assert-current-schema-id"
+          - "assert-last-assigned-partition-id"
+          - "assert-default-spec-id"
+          - "assert-default-sort-order-id"
```

Comment:
Minor: it is not a big deal, but it would be more readable if we move the `discriminator` under the properties section. Spec reader will understand better about the mapping between the `type` property name and correspondent schema.

Reply:
Done.